src/Main.cpp

```diff
@@ -299,7 +299,7 @@ int main(int argc, char** argv)
                           << state.magnetic_info.magnetometers[0] << " "
                           << state.magnetic_info.magnetometers[1] << " "
                           << state.magnetic_info.magnetometers[2] << " "
-                          << atan2(state.magnetic_info.magnetometers[2],
+                          << atan2(state.magnetic_info.magnetometers[1],
                                  state.magnetic_info.magnetometers[0]) *
                                  rad2deg
```

Comment:
What is the difference between this value and the one in `measured_euler_angles[2]` ?

Reply:
This one is calculated here, the other one comes from the IMU.